### PR TITLE
Blockchain Presence

### PR DESCRIPTION
**Pull request by TheGamerzs**
image
image
image

### CODE REVIEW

File: websites/B/Blockchain/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "url": [
+    "www.blockchain.com",
+    "login.blockchain.com",
+    "exchange.blockchain.com",
+    "support.blockchain.com",
+    "www.blockchain-status.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/tz8fOEn.png",
```
Comment (Bas950) [inline websites/B/Blockchain/dist/metadata.json:18]: upload a logo of at least 512x512 size

File: websites/B/Blockchain/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "thumbnail": "https://i.imgur.com/tz8fOEn.png",
+  "color": "#3d89f5",
```
Comment (Bas950) [inline websites/B/Blockchain/dist/metadata.json:20]: thumbnail should be a promotional banner or screenshot of the website homepage.

File: websites/B/Blockchain/presence.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
+var presence = new Presence({
+  clientId: "776522940517974016"
+});
+
+var browsingStamp = Math.floor(Date.now() / 1000);
+var currencyList = [
```
Comment (Bas950) [inline websites/B/Blockchain/presence.ts:6]: bro really.... var?

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by TheGamerzs** [comment]
Fixed

File: websites/B/Blockchain/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "en": "The World's Most Popular Way to Buy, Hold, and Use Crypto"
+  },
+  "url": [
+    "www.blockchain.com",
+    "login.blockchain.com",
+    "exchange.blockchain.com",
+    "support.blockchain.com",
+    "www.blockchain-status.com"
+  ],
+  "version": "1.0.3",
```
Comment (Bas950) [inline websites/B/Blockchain/dist/metadata.json:17]: version 1.0.0 on release

File: websites/B/Blockchain/presence.ts [hunk 5]
```diff
@@ -0,0 +1,134 @@
+const presence = new Presence({
+  clientId: "776522940517974016"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/B/Blockchain/presence.ts:5]: merge

File: websites/B/Blockchain/presence.ts [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+let currencyTitle: string;
+let pageStatus: string;
```
Comment (Bas950) [inline websites/B/Blockchain/presence.ts:7]: merge

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/B/Blockchain/presence.ts (hunk 5 above)
Comment (TheGamerzs) [reply websites/B/Blockchain/presence.ts:5]: wdym by this?

Comment (ririxidev) [reply websites/B/Blockchain/presence.ts:5]: bruh.
e.g.
```javascript
const d = f(),
g = h();
```

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**